### PR TITLE
Removed unnecessary debug statements, Cyclopes attack dynamics improved

### PR DESCRIPTION
**Pull request by heostar**
- Cyclopes distance to player is now calculated by the SphereCollider fixed to the Cyclopes body
- Both attack types now makes much more sense visually than before.

### CODE REVIEW

**Review by thedanhle — APPROVED** [review]
looks great!

File: Fluffed Up/Assets/Scripts/WaveManager.cs [hunk 1]
```diff
@@ -381,12 +389,15 @@ void ExecuteAttack(EnemyBase enemy, float damage, bool playerAttacks)
...
                 bool withinAngle = math.abs(Vector3.Angle(enemy.transform.forward, player.transform.position - enemy.transform.position)) < 30 ;
-                if (withinDistance && withinAngle)
+                if (withinAngle)
                 {
                     player.TakeDamage(damage, 0);
                 }
+                else
+                {
+                    Debug.LogError("Out of range");
+                }
```
Comment (thedanhle) [inline Fluffed Up/Assets/Scripts/WaveManager.cs]: Maybe remove this debug

File: Fluffed Up/Assets/Scripts/WaveManager.cs [hunk 2]
```diff
@@ -144,6 +144,14 @@ private IEnumerator ComputeOnslaught()
...
         Debug.Log("Computing Onslaught");
 
+        if (currentWave == 0)
+        {
+            nextEnemyQueue.Clear();
+            nextEnemyQueue = new Queue<string>(new List<string>() { "BossCyclopes" });
+            Debug.Log("Onslaught Computed");
+            yield break;
+        }
+
```
Comment (thedanhle) [inline Fluffed Up/Assets/Scripts/WaveManager.cs:154]: making sure we don't accidently merge this

File: Fluffed Up/Assets/Scripts/WaveManager.cs (hunk 1 above)
Comment (heostar) [reply Fluffed Up/Assets/Scripts/WaveManager.cs]: Thanks for catching that

**Pull request merged**